roadie-rails.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "roadie", ">= 3.1", "< 5.0"
 
   spec.add_development_dependency "bundler", "~> 2.2"
-  spec.add_development_dependency "rails", ">= 5.1", "< 7.0"
+  spec.add_development_dependency "rails", ">= 5.1", "< 8.0"
```

Comment:
Normally we'd go for `< 7.1` because it could break in version 7.1 as Rails do not follow semantic versioning and we're using private APIs.